projects/whydense/cifar/cifar_run.py

```diff
@@ -34,7 +34,7 @@
 def train_models(configs, project_dir):
     """Run all the training experiments specified in configs."""
     # download dataset
-    data_dir = os.path.join(project_dir, "data")
+    data_dir = os.path.join(os.environ["HOME"], "nta", "data")
```

Comment:
"nta" is specific to our laptops, it may not work everywhere.

Reply:
It's also on AWS. This way it's convenient since the exact paths are different between my laptop and AWS. Also, it's only the default if you don't specify in config.  I pushed a change to make it clearer.